ss_rules.json

```diff
@@ -0,0 +1,792 @@
+					"Section-Series-Page (ssp)",
+					"Section-Halftitle (sht)",
+					"Section-Titlepage (sti)",
+					"Section-Copyright-Page (scr)",
+					"Section-Dedication (sde)",
+					"Section-Contents (stc)",
+					"Section-Introduction-Frontmatter (sif)",
+					"Section-Preface (spf)",
+					"Section-Foreword (sfw)",
+					"Section-Introduction-Chapter (sic)",
+					"Section-Part (spt)",
+					"Section-Chapter (scp)",
+					"Section-Appendix (sap)",
+					"Section-Acknowledgments (sak)",
+					"Section-Afterword (saw)",
+					"Section-Conclusion (scl)",
+					"Section-About-Author (saa)",
+					"Section-Back-Ad (sba)",
+					"Section-Epigraph (sep)",
+					"Section-Excerpt-Opener (seo)",
+					"Section-Excerpt-Chapter (sec)",
+					"Section-Notes (snt)",
+					"Section-Bibliography (sbi)",
+					"Section-Glossary (sgl)",
+					"Section-Index (sin)"
+				]
+			}
+		},
+		"contiguous_block_criteria_02": {
+			"multiple":	"False",
+			"styles": [
+				"FM Author Name (fmau)",
+				"FM Author Signature (ausig)",
+				"FM Dramatis Personae (fmdp)",
+				"FM Epigraph - non-verse (fmepi)",
+				"FM Epigraph - verse (fmepiv)",
+				"FM Epigraph Source (fmeps)",
+				"FM Level-1 Subhead (fmh1)",
+				"FM Level-2 Subhead (fmh2)",
+				"FM Level-3 Subhead (fmh3)",
+				"FM Level-4 Subhead (fmh4)",
+				"FM Level-5 Subhead (fmh5)",
+				"FM Subhead (fmsh)",
+				"FM Text (fmtx)",
+				"FM Text ALT (afmtx)",
+				"FM Text No-Indent (fmtx1)",
+				"FM Text No-Indent ALT (afmtx1)",
+				"FM Text No-Indent Space After (fmtx1#)",
+				"FM Text No-Indent Space Around (#fmtx1#)",
+				"FM Text No-Indent Space Before (#fmtx1)",
+				"FM Text Space After (fmtx#)",
+				"FM Text Space Around (#fmtx#)",
+				"FM Text Space Before (#fmtx)"
+			],
+			"previous_sibling": {
+				"required_styles": [
+					"Section-Ad-Card (sac)",
+					"Section-Front-Sales (sfs)",
+					"Section-Series-Page (ssp)",
+					"Section-Halftitle (sht)",
+					"Section-Titlepage (sti)",
+					"Section-Copyright-Page (scr)",
+					"Section-Dedication (sde)",
+					"Section-Contents (stc)",
+					"Section-Introduction-Frontmatter (sif)",
+					"Section-Preface (spf)",
+					"Section-Foreword (sfw)",
+					"Section-Introduction-Chapter (sic)",
+					"Section-Part (spt)",
+					"Section-Chapter (scp)",
+					"Section-Appendix (sap)",
+					"Section-Acknowledgments (sak)",
+					"Section-Afterword (saw)",
+					"Section-Conclusion (scl)",
+					"Section-About-Author (saa)",
+					"Section-Back-Ad (sba)",
+					"Section-Epigraph (sep)",
+					"Section-Excerpt-Opener (seo)",
+					"Section-Excerpt-Chapter (sec)",
+					"Section-Notes (snt)",
+					"Section-Bibliography (sbi)",
+					"Section-Glossary (sgl)",
+					"Section-Index (sin)",
+					"FM Head (fmh)",
+					"FM Head ALT (afmh)",
+					"FM Head Nonprinting (fmhnp)",
+					"FM Title (fmt)"
+				]
+			}
+		}
+	},
+	"Section-Foreword (sfw)": {
+		"section_type":	"frontmatter",
+		"contiguous_block_criteria_01": {
+			"multiple":	"False",
```

Comment:
There actually could be multiple forewords. Sometimes books will have multiple people add a foreword, e.g., "Foreword by Charles Barkely" and also "Foreword by Michael Jordan".